etl/etl_target.py

```diff
@@ -118,7 +120,6 @@ def target_find():
     sql_command = "SELECT bs.nct_id, CONCAT(bs.description, ' ',  dd.description) FROM ctgov.brief_summaries bs INNER JOIN ctgov.detailed_descriptions dd ON bs.nct_id = dd.nct_id"
     print(f"Run command: {sql_command}")
```

Comment:
I know this wasn't edited in this commit but, in my experience, the prints don't seem to show up when run from a Fargate Task unless we add "flush=True" so we should try to add that to all the print statements.

Reply:
kk adding